usr/share/ublue-os/just/custom.just

```diff
@@ -122,35 +131,37 @@ jetbrains-toolbox:
   echo "Launching JetBrains Toolbox"
   ./jetbrains-toolbox-$BUILD_VERSION/jetbrains-toolbox
 
+# Install nix and Devbox
 nix-devbox:
   echo 'Setting phasers to kill. Installing nix.'
   curl -s https://raw.githubusercontent.com/dnkmmr69420/nix-installer-scripts/main/installer-scripts/silverblue-nix-installer.sh | bash
   echo 'Installing devbox!'
   curl -fsSL https://get.jetpack.io/devbox | bash
   echo 'You MUST reboot to continue'
 
+# Install nix and Devbox (Global Profile)
 nix-devbox-global:
   echo 'Installing devbox global profile.'
   devbox global pull https://devbox.getfleek.dev/high
   echo 'run "devbox global run install-bash-hook" to configure bash shell'
   echo 'run "devbox global run install-zsh-hook" to configure zsh shell'
   echo 'run "devbox global run" to see other available configuration commands'
 
-tea:
```

Comment:
Was this intended?

Reply:
Yeah I tried it for a bit but feels like we have more established things in here. We could leave it, just feels like it's not really picking up steam?

Reply:
Let's clean it up then